docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+services:
+  waveform-controller:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        HTTP_PROXY: ${HTTP_PROXY}
+        http_proxy: ${http_proxy}
+        HTTPS_PROXY: ${HTTPS_PROXY}
+        https_proxy: ${https_proxy}
+    # ideally we'd use docker secrets but it's not enabled currently
+    env_file:
+     - ./config/settings.env
+    volumes:
+      - ./waveform_data:/app/waveform_data
```

Comment:
The waveform-reader already mounts a directory called `waveform-saved-messages`. For consistency and clarity, I think this directory should be called `waveform-export` (hyphen, not underscore) or something similar to differentiate it.